### DISCUSSION TITLE
HA background activities: Display information in Frontend (statusbar, etc.)

### DISCUSSION THREAD

**Discussion by erd82** [home-assistant/frontend#9789] [Supervisor]
**Wouldn't it make sense to see information about running background activities in HA (like snapshots, updates, etc.) where you can track what happens in the background?**

Didn't you sometimes have the same problems?
* When creating a snapshot I was often wondering later if I really triggered the snapshot from Supervisor UI. I had to check the logs again and again if the snapshot creation is really in process. Suddenly the new snapshot appeared. :smirk:
* When updating an add-on the UI just showed the busy indicator / hourglass (or how is it called in English?). But you don't exactly know the status for some time and it seems that nothing is happening.

I trust that there are also other significant background activities in HA which cause quite some load on the hardware.

Opportunities, if you can track the background activities:
* you can offer a nice UI in frontend to see all activities at a glance.
* within the relevant application (e.g. Supervisor > Snapshots) you can display additional information about current background activity, such as status.
* you can also offer to display the logs only related to this background activity (e.g. for troubleshooting).

See also https://community.home-assistant.io/c/feature-requests/13

**Comment by ludeeus** [comment]
Needs https://github.com/home-assistant/supervisor/issues/1946
